projects/brotli-java/FuzzDecode.java

```diff
@@ -0,0 +1,37 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////////
+
+import com.code_intelligence.jazzer.api.FuzzedDataProvider;
+
+import java.io.*;
+import org.brotli.dec.BrotliInputStream;
+
+public class FuzzDecode {
+  public static void fuzzerTestOneInput(FuzzedDataProvider data) {
+    byte[] buffer = new byte[65536];
+    ByteArrayInputStream input = new ByteArrayInputStream(data.consumeBytes(65536));
+    ByteArrayOutputStream output = new ByteArrayOutputStream();
+    try {
+      BrotliInputStream brotliInput = new BrotliInputStream(input);
+      while (true) {
+        int len = brotliInput.read(buffer, 0, buffer.length);
```

Comment:
Since this is a roundtrip, would be nice to check input == output

Reply:
This is only a decompression test, right? i.e. the output should only be equal to the original if we compress it back, which we don't atm (the output variable was unused in fuzzer - I removed it now too so it's no longer there). I can do a roundtrip test too though, but let me do that a bit later when I too get free cycles